docs_sphinx/introduction/preferences.rst

```diff
@@ -15,29 +15,28 @@ sparse matrix methods ('SPARSE'). You can set the preference like this::
 
     prefs.devices.genn.connectivity = 'DENSE'
 
+On Windows, Brian2GeNN will try to find the file ``vcvarsall.bat`` to enable
+compilation with the MSVC compiler automatically. If this fails, or if you have
+multiple versions of MSVC installed and want to select a specific one, you can
+set the ``codegen.cpp.msvc_vars_location`` preference.
 
-Compiler preferences
+CUDA preferences
 --------------------
-Brian2GeNN will use the compiler preferences specified for Brian2 for the
```

Comment:
did we conclude that the option to set extra compiler arguments is no longer needed? I can't recall what exactly the use case was but I am sure we had a reason why we added these?

Reply:
The windows ones definitely don't work any more but I can check I didn't break the *nix ones and reinstate that bit of documentation